### ISSUE TITLE
What about default locale?

### ISSUE THREAD

**Issue by idoo** [tmpvar/jsdom#1489]
Is here ways for change default locale for tests?

I ask about emulations ```window.navigator.language``` or something like this.
For example ```toLocaleString``` or ```toLocaleDateString``` using as default browser locale

```
var date = new Date(Date.UTC(2012, 11, 20, 3, 0, 0));
console.log(date.toLocaleTimeString());

```

**Comment by Sebmaster** [comment]
While we could theoretically change window.navigator.language, stuff like toLocaleTimeString is (normally) out of reach. We could maybe overwrite the formatting functions of Date in the sandbox and replace them with `Intl.DateTimeFormat` calls, but that'd require you to build node with multiple locales (or we could maybe shim with an optional install of [intl](https://www.npmjs.com/package/intl)).

All of those don't sounds like great course of action though (except making navigator.language configurable).

**Comment by domenic** [comment]
You can override navigator.language with Object.defineProperty.

**Comment by BrunoQuaresma** [comment]
@domenic override the language property not works.

**Comment by domenic** [comment]
Yes, that was explained to you in https://github.com/tmpvar/jsdom/issues/1489#issuecomment-220304329

**Comment by BrunoQuaresma** [comment]
@domenic  I'm using this code.:
`date.toLocaleDateString('pt-BR')`

In my chrome browser the result is.: 9/13/2016
When I run the tests in local.: 2016-09-13
When I runt the tests in Circle CI.: 9/13/2016

Do you have any ideia why it happens?

**Comment by domenic** [comment]
I don't, sorry; I've never used multiple locales in Node.js.

**Comment by BrunoQuaresma** [comment]
Ok, thanks again!
